src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -100,7 +100,7 @@ def _store(
         cached_replay_recording_segment: CachedAttachment,
     ) -> None:
         with sentry_sdk.start_transaction(
-            op="replays.consumer.flush_batch", description="Replay recording segment stored."
+            op="replays.consumer.flush_batch", name="replays.consumer.flush_batch"
```

Comment:
There's not really a point in setting the op and name to be the same here. You could set the op to `replays.consumer` so that you can filter for `transaction.op:replays.consumer` to find all of these transactions.

Reply:
👍🏼